### PR TITLE
TELCODOCS#1864: Test and validate telco features on AMD CPUs (Genoa and Bergamo)

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.18

Issue:
- [TELCODOCS-1864](https://issues.redhat.com/browse/TELCODOCS-1864)
- [TELCODOCS-1865](https://issues.redhat.com/browse/TELCODOCS-1865)

Link to docs preview:
- [Node Tuning Operator](https://85373--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-du-components.html#telco-ran-node-tuning-operator_ran-ref-design-components)
- [Telco core RDS engineering considerations](https://85373--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/telco_ref_design_specs/core/telco-core-rds-use-cases#telco-core-ref-eng-usecase-model_ran-core-design-overview)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@MarSik 
Could you please review this PR?
Thanks!

File: modules/telco-core-ref-eng-usecase-model.adoc [hunk 1]
```diff
@@ -9,8 +9,13 @@
 The following engineering considerations are relevant for the telco core common use model.
 
 Worker nodes::
-* Worker nodes should run on Intel 3rd Generation Xeon (IceLake) processors or newer.
+* Worker nodes should run on either of the following CPUs:
+
+** Intel 3rd Generation Xeon (Ice Lake) 2.20 GHz CPUs or newer
+** AMD EPYC Zen 4 CPUs (Genoa, Bergamo, or newer)
 +
+The AMD EPYC Zen 4 CPUs (Genoa, Bergamo or newer) are fully supported. However, the power consumption evaluations are ongoing. It is recommended to evaluate features, such as per pod power management, to determine any potential impact.
```
Comment (MarSik) [inline modules/telco-core-ref-eng-usecase-model.adoc:17]: I think you are missing the [NOTE] designator here.

Comment (sr1kar99) [reply modules/telco-core-ref-eng-usecase-model.adoc:17]: I intentionally did not use a note here as there is another note next to this para. As per the the style guide, it is not recommended to stuff two notes one after the other. @MarSik

**Comment by sr1kar99** [comment]
@mrniranjan @imiller0 
Could you please review this PR?
Thanks!

**Review by skopacz1 — APPROVED** [review]
LGTM!

**Comment by aireilly** [comment]
I think we should get this doc update into https://gitlab.cee.redhat.com/reference-configurations/reference-design-specifications so that we can do a clean PR for all the RDS 4.18 content in openshift-docs. @sr1kar99 let's chat Monday.

**Pull request closed without merging**